setup.py

```diff
@@ -136,7 +136,7 @@ def pkgPath(root, path, rpath="/"):
     version=version,
     url='https://github.com/python-diamond/Diamond',
     author='The Diamond Team',
-    author_email='https://github.com/python-diamond/Diamond',
+    author_email='pypi@josediazgonzalez.com',
```

Comment:
I'm happy to add an e-mail but it'd be nice if GitHub or pypi supported some sort of proxy e-mail address or perhaps point to a mailing list instead?